core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.html

```diff
@@ -7,6 +7,11 @@
            class="progress">
         <div class="progress-bar bg-success"
              role="progressbar"
+             [attr.aria-label]="progressBarLabel"
+             title="progressBar"
+             ngbTooltip="{{opportunity?.translationsCount}}/{{opportunity?.totalCount}} cards accepted"
+             container="body"
+             placement="bottom"
```

Comment:
Are these even necessary here? Also, note that the tooltip description is incorrect since this is the progress bar for practice questions not translations.

Reply:
The reason i added the ngbtooltip was due to *ngIf is a structural directive *ngIf="!translationProgressBar" due to which i have pasted the same tooltip below  that was used in the codebase  i should have made it ngbTooltip="{{opportunity?.progressPercentage}}% of practice questions completed" something anyway 
i will remove this line and make a commit Thanks @sagangwee